### DISCUSSION TITLE
SQL Repo not being recognized

### DISCUSSION THREAD

**Discussion by arnonrdp** [github-linguist/linguist#5900] [Classification] [answered]
I recently added a repository full of `.sql` files to GitHub.

But strangely no file was recognized as any sql language type.

So I added a `.gitattributes` and set `*.sql linguist-detectable=true` but it didn't work.

Repo: https://github.com/arnonrdp/Curso-SQL

**Comment by lildude** [comment, marked as answer]
Changes to `.gitattributes` alone is not always sufficient to force a re-analysis. Pushing another change to another file should trigger an analysis.

**Reply by arnonrdp** [reply]
I made a small change to a `.sql` file directly on GitHub, but it doesn't seem to work.

I just updated my thread with the repository.

**Reply by lildude** [reply]
You've got a typo in the filename:

![CleanShot 2022-05-18 at 14 16 06@2x](https://user-images.githubusercontent.com/627280/169047799-4723076b-b5eb-4570-b97d-4863b28fffde.png)

**Reply by arnonrdp** [reply]
Haha ha
I can not believe this!
Thanks for letting me know, man.
I really appreciate.
